api/sheets/get_cohort_members.md

```diff
@@ -6,22 +6,24 @@ parent: Google Sheets
 grand_parent: API
 ---
 
-# Get the values of a given property for a node.
+# Retrieve the values of a given property for a node
```

Comment:
Same comment on the page title

Reply:
Discussion: https://github.com/datacommonsorg/docsite/pull/127#discussion_r571358818